bin/ycsb

```diff
@@ -20,6 +20,11 @@ COMMANDS = {
         "description" : "Execute the load phase",
         "main"        : "com.yahoo.ycsb.Client",
     },
+    "delete" : {
+        "command"     : "-delete",
+        "description" : "Execute the clean-up phase",
```

Comment:
I would not call this the "clean-up" phase since there is actually a "CLEANUP" phase that shows up in the latency metrics. Maybe "Delete all data added during the load phase".